zigpy/device.py

```diff
@@ -136,6 +153,39 @@ def schedule_initialize(self) -> asyncio.Task | None:
 
         return self._initialize_task
 
+    async def discover_ieee(self) -> EUI64:
+        self.info("Requesting IEEE address")
+
+        status, ieee, nwk, *_ = await self.zdo.IEEE_addr_req(
+            self.nwk, 0x00, 0x00, tries=2, delay=0.1
```

Comment:
the idea for these devices that only one address is unknown: either IEEE or NWK, but not both. And most of the times I'm expecting it  to be the IEEE address. If device is on the Network, then coordinator/Network most likely to know the network address and this would be triggered, when radio libarry fails to find a device in `get_device(nwk=received_frame_source_address)`